.github/fabricbot.json

```diff
@@ -860,6 +860,75 @@
         ],
         "taskName": "Author Response needed"
       }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "IssueCommentResponder",
+      "version": "1.0",
+      "id": "mgZxIoqeF7GWUo7-t3773",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "name": "commentContains",
+              "parameters": {
+                "commentPattern": "\\/loc",
```

Comment:
consider: this will match any substring of any comment that contains `/loc`, such as "we are having a build/localization problem"

You might want `\\/loc\\b` or similar.